### DISCUSSION TITLE
Java-19 Amazon Corretto Errors

### DISCUSSION THREAD

**Discussion by asimumba** [airbytehq/airbyte#30078] [Connector Questions]
Hi All,

I have the following errors showing `-XX:+ExitOnOutOfMemory` error from the Airbyte app. This results in the memory going all the way to 100% and the server becomes unresponsive. How do I prevent Airbyte from consuming too much memory?

**Comment by marcosmarxm** [comment]
@asimumba can you give more information about what you're trying to do or where are you trying to deploy Airbyte? Would be nice to update the title of your issue too.
